gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/tasks/CreateManifestTaskTest.groovy

```diff
@@ -45,22 +45,4 @@ class CreateManifestTaskTest extends ProjectSpec {
         Exception exception = thrown()
         exception.message.contains("Project version must be a valid SLS version: 1.0.0foo")
     }
-
-    def "Fails if user declares dependency on the same product"() {
```

Comment:
Was there a reason to get rid of this test?

Reply:
I didn't its just been moved https://github.com/palantir/sls-packaging/blob/e9da416f13ca49761986ba9d261cbc1c6373df68/gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/pdeps/ResolveProductDependenciesTaskSpec.groovy#L78

Reply:
it's an existing test, just migrated.  I presume it's there because it's redundant (how can you have a dependency on your self?) and may indicate a typo.